specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/serverAutomaticTuning.json

```diff
@@ -0,0 +1,344 @@
+          }
+        },
+        "x-ms-examples": {
+          "Get a database's automatic tuning settings": {
+            "$ref": "./examples/DatabaseAutomaticTuningGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "DatabaseAutomaticTuning"
+        ],
+        "description": "Update automatic tuning properties for target database.",
+        "operationId": "DatabaseAutomaticTuning_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested automatic tuning resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatabaseAutomaticTuning"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the specified database automatic tuning settings.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAutomaticTuning"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidIdentifier - The identifier contains NULL or an invalid unicode character.\n\n * 400 InvalidAutomaticTuningUpsertRequest - The update automatic tuning request body does not exist or has no properties object.\n\n * 400 InvalidAdvisorAutoExecuteStatus - Specified auto-execute status for the advisor is not allowed.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 DatabaseDoesNotExist - The requested database was not found\n\n * 404 SubscriptionDoesNotHaveElasticPool - The requested elastic pool was not found\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 405 NotSupported - This functionality is not supported.\n\n * 409 Conflict - Request could not be processed because of conflict in the request.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation.\n\n * 503 ServiceTemporarilyUnavailable - Feature temporarily unavailable."
+          }
+        },
+        "x-ms-examples": {
+          "Updates database automatic tuning settings": {
+            "$ref": "./examples/DatabaseAutomaticTuningUpdate.json"

```

Comment:
This file doesn't exist.

Reply:
Somehow I added databases automatic tuning swagger here instead of server automatic tuning swagger, I committed proper server automatic tuning swagger now.Leave a comment